src/mem/cache/cache_blk.hh

```diff
@@ -545,10 +547,10 @@ class TempCacheBlk final : public CacheBlk
     }
 
     void
-    insert(const Addr addr, const bool is_secure) override
+    insert(const KeyType &tag) override
     {
-        CacheBlk::insert(addr, is_secure);
-        _addr = addr;
+        CacheBlk::insert(tag);
+        _addr = tag.address;
```

Comment:
We do store the secure bit.. If you see the tag type is KeyType which carries the secure bit. CacheBlk::insert ultimately calls the TaggedEntry::insert which stores the secure bit in it.

Reply:
Indeed. Resolved.

side note: I have no idea how to mark things as resolved in GitHub - can one only do that as a maintainer?

Reply:
I don't know tbf. Probably. This is something I don't like about GitHub, together with the fact that I cannot add a non-maintainer as reviewer

Reply:
Gonna vent a bit too ;)

Although I understand why we adopted GitHub, the move from gerrit was probably the main reason that made me decide to step out of the maintainer role. The UI here is extremely confusing to me, and I keep losing track of the requests I made. Besides, finding what changed between reviews is usually a struggle; right now, for example, if I click to ´show changes since my last review´ I get a ´We went looking everywhere, but couldn’t find those commits.´.